Cargo.toml

```diff
@@ -129,6 +143,23 @@ build-override.opt-level = 3
 [profile.dev]
 build-override.opt-level = 3
 
+# The profile that 'dist' will build with
+[profile.dist]
+inherits = "release"
+lto = "thin"
+
+[[bin]]
+name = "neptune-core"
+path = "src/main.rs"
+
+[[bin]]
+name = "neptune-cash-tui"
```

Comment:
I think we should be consistent with the naming.

neptune-core, one hyphen.  neptune-cash-tui, two hyphens.

I vote for neptune-<core|tui|cli>

or failing that:

neptune-cash-<core|tui|cli>